src/resource/user/user.service.ts

```diff
@@ -17,4 +21,23 @@ export class UserService {
   async findByUsername(_username: string) {
     return this.prisma.user.findFirst({ where: { username: _username } });
   }
+
+  async createPasswordResetLink(_id: string) {
+    const { createHmac, randomBytes } = await import('crypto');
+    const secret = process.env.CRYPTO_SECRET;
+    const token = randomBytes(32).toString('hex');
+    const resetToken = createHmac('sha256', secret).update(token).digest('hex');
+    const resetTokenExpires = addHours(new Date(), 1).toISOString();
+    console.log(resetTokenExpires);
```

Comment:
So instead of `console.log`, let's use 
```
const { Logger } from '@nestjs/common'

Logger.log()
```

No particular reason, it just looks nicer and has more info.

Reply:
Also, we don't want to log the expiry out. We just need to log the operation has success or log error if there's anything wrong with it.

Reply:
This has been updated